.github/workflows/pr-lint.yaml

```diff
@@ -3,22 +3,40 @@ on: pull_request
 jobs:
   link-ticket:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
     steps:
       - name: Check ticket name conforms to requirements
-        run: echo ${{ github.event.pull_request.head.ref }} | grep -i -E -q "((apm|niad|AMB)-[0-9]+)|(dependabot\/)"
+        run: echo ${{ github.event.pull_request.head.ref }} | grep -i -E -q "((apm|niad|AMB|PI|amp)-[0-9]+)|(dependabot\/)"
```

Comment:
Whats amp?

Reply:
Previous project. Just adding it as it was added in a previous PR that never got merged